### PR TITLE
feat: OPTIC-3: Enable users to submit and pause their lead time by returning to the dm list 

### PR DESCRIPTION
**Pull request by robot-ci-heartex**
Hi @Travis1282!

This PR was created in response to a PRs in upstream repo:
- https://github.com/HumanSignal/dm2/pull/225
- https://github.com/HumanSignal/label-studio-frontend/pull/1541

### CODE REVIEW

File: docs/source/tags/taxonomy.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Use with the following data types: audio, image, HTML, paragraphs, text, time se
 | [apiUrl] | <code>string</code> |  | URL to fetch taxonomy from remote source; API should accept optional array `path` param: `apiUrl?path[]=root&path[]=child1` to return only nested children of `child1` node[^2] |
 | [leafsOnly] | <code>boolean</code> | <code>false</code> | Allow annotators to select only leaf nodes of taxonomy |
 | [showFullPath] | <code>boolean</code> | <code>false</code> | Whether to show the full path of selected items |
-| [pathSeparator] | <code>string</code> | <code>&quot;/&quot;</code> | Separator to show in the full path (default is " / ") |
```
Comment (bmartel) [inline docs/source/tags/taxonomy.md]: How come this changed?

**Review by bmartel — APPROVED** [review]

**Comment by robot-ci-heartex** [comment]
To enable Auto Merge for this PR also merge those PRs:
- https://github.com/HumanSignal/dm2/pull/225

File: docs/source/tags/taxonomy.md (hunk 1 above)
Comment (hlomzik) [reply docs/source/tags/taxonomy.md]: this happens when corresponding LS PR for another ticket was not merged for some reason and those changes from LSF PR are spilled to all the further PRs until some of them got merged in

**Comment by hlomzik** [comment]
related changes are already merged long time ago

**Pull request closed without merging**